src/App.js

```diff
@@ -1,5 +1,44 @@
+import Car from './Car';
+import MessageProcessor from './MessageProcessor';
+import Validator from './Validator';
+
 class App {
-  async play() {}
+  constructor() {
+    this.cars = [];
+    this.tryCount = 0;
+  }
+
+  progressRace() {
+    MessageProcessor.result();
+    for (let i = 0; i < this.tryCount; i += 1) {
+      this.cars.forEach(car => {
+        const carCondition = Car.getCondition();
+        car.move(carCondition);
+      });
```

Comment:
`Car.getCondition()`을 변수에 할당하지 않고 `car.move()`에 바로 인수로 전달해주는 것은 어떨까요?

변수에 담지 않더라도 함수를 호출하는 것뿐이라 어렵지 않게 코드를 해석할 수 있고,
아래처럼 더 깔끔하게 수정해볼 수 있을 것 같습니다.

```suggestion
      this.cars.forEach(car => car.move(Car.getCondition()));
```